runtime/oti/VMHelpers.hpp

```diff
@@ -2068,7 +2129,13 @@ class VM_VMHelpers
 	static VMINLINE bool
 	isContinuationMounted(J9VMContinuation *continuation)
 	{
-		return (NULL != continuation->carrierThread);
+		return J9_ARE_ANY_BITS_SET(continuation->state, ~(uintptr_t)J9_GC_CONTINUATION_CONCURRENTSCANNING);;
+	}
+
+	static VMINLINE bool
+	isContinuationMountedOrConcurrentlyScanned(J9VMContinuation *continuation)
```

Comment:
Consider moving those 2 new helpers to more specific ContinuationHelper.hpp. As I said before, even the other existing ones that are continuation specific belong there, but for easier code review, it's probably better to leave them here.

Reply:
Consider same for any other NEW methods in this file